schemas/PositiveFiniteAmount.json

```diff
@@ -0,0 +1,8 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "id": "PositiveFiniteAmount",
+  "title": "PositiveFiniteAmount",
+  "description": "A string representation of a positive, finite number",
+  "type": "string",
+  "pattern": "^(?!(?:^[-+]?[0.]+(?:[Ee]|$)))(?!(?:^-))(?:(?:[+-]?)(?=[0123456789.])(?:(?:(?:[0123456789]+)(?:(?:[.])(?:[0123456789]*))?|(?:(?:[.])(?:[0123456789]+))))(?:(?:[Ee])(?:(?:[+-]?)(?:[0123456789]+))|))$"
```

Comment:
This looks really complex. What about using https://github.com/ripple/ripple-lib/blob/develop/src/common/schemas/objects/value.json#L7 ?